### ISSUE TITLE
Request for Detailed Hyperparameters to Reproduce Paper Results

### ISSUE THREAD

**Issue by chanztuying** [rjha18/vec2vec#16]
Hi,

Thank you for releasing this codebase! I am trying to reproduce the results reported in your paper, but I am running into issues with validation performance, especially on the top-k accuracy metrics.

  I am using the unsupervised.toml configuration as the base.
  The only parameters I changed are:
```
  n: 1000000
  u: gte
  s: gtr
  d: nq
  u: True
  e: 200
```
  All other parameters are as in unsupervised.toml.

Despite following the above setup, I am getting poor results for the validation top-k metrics (e.g., val/top_1_acc, val/top_k_acc). 

Could you please provide a more detailed list of hyperparameters and settings (including num_points, data splits, or other relevant details) that were used to produce the results in the paper? 

Thank you for your help!

**Comment by rjha18** [comment]
Hey there and thanks for the interest in our project!

Unfortunately, our current GAN-based implementation is quite unstable (something we’re working on), so for some encoder pairings things like start seed and weight initializations really matter. This is *especially* true of cross-backbone translations like the one in `unsupervised`. I would recommend trying one of the following:
1. More seeds for cross-backbone models
2. If you're just looking to get something off the ground try the new unsupervised.toml (which translates within the same backbone)
3. Use the pre-trained models / configs in **Releases**. These are the actual checkpoints (and configs used to train them :)) we used for the paper!

Some details: all our experiments were run on single NVIDIA A100s, A40s/A6000s, L4s, and 2080tis. Initialization is *not* consistent across GPU, so changing GPUs is the equivalent of changing seeds! We try a few different seeds for each experiment. Same backbone models tend to be much more stable than cross-backbone translations. 2080tis only support fp16, so any experiments were set accordingly.
